docs/reference/index.md

```diff
@@ -386,6 +386,45 @@ The values for each key currently match its default.
 
 # DROPBOX_REFRESH_TOKEN=""
 
+########### GOOGLE DRIVE STORAGE
+
+# The JSON credentials for a Google service account with access to Google Drive. The value should be the path to a json file.
```

Comment:
This is correct, but judging from the issues I've seen raised in this repo over time, I think it wouldn't hurt to also mention this file needs to be mounted into the container.